src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -427,7 +427,9 @@ class ActivityFeed extends React.Component<Props, State> {
         const { file } = this.props;
         const { restored_from, modified_at, file_version } = file;
 
-        if (restored_from) {
+        // Ensures restored version is only added on first feed loads
+        const lastVersion = versions.total_count ? versions.entries[versions.total_count - 1] : null;
```

Comment:
Maybe change to : {} instead of null so the checking of lastVersion is not needed below?